httpbp/prometheus.go

```diff
@@ -95,6 +95,7 @@ var (
 var (
 	clientLatencyLabels = []string{
 		methodLabel,
+		endpointLabel,
```

Comment:
we are only adding the label to the metrics definition but never set it to non-empty?

Reply:
I've added the metrics to the call sites with empty values